src/modules/github.cpp

```diff
@@ -68,10 +74,13 @@ namespace modules {
     try {
       content = request();
     } catch (application_error& e) {
-      m_log.warn("%s: cannot complete the request to github: %s", name(), e.what());
+      m_log.info("%s: cannot complete the request to github: %s", name(), e.what());
```

Comment:
```suggestion
      if (!m_offline) {
        m_log.info("%s: cannot complete the request to github: %s", name(), e.what());
      }
```

I think we should only show the message when we are no longer online, otherwise it would spam the logs for offline users.